client/src/pages/MemberProfilePage/MemberProfilePage.js

```diff
@@ -46,7 +46,8 @@ class MemberProfilePage extends Component {
   render() {
     let people = [
       {
-        name: "Suzie",
+        userFirstName: "Suzie",
+        userSurname:"Carmicheal",
```

Comment:
If this is for the user, then the properties don't need `user` in them (it's redundant)